test/performance/odp_pool_perf.c

```diff
@@ -645,6 +648,31 @@ static void print_stat(test_global_t *global)
 	       (1000.0 * frees_ave) / nsec_ave);
 	printf("  events per sec:       %.3f M\n\n",
 	       (1000.0 * events_ave) / nsec_ave);
+
+	printf("TOTAL events per sec:       %.3f M\n\n",
```

Comment:
Couple extra spaces. Print would look nicer if the value would be aligned with the previous ones.